src/sbuf_flex_scanner.h

```diff
@@ -85,7 +86,8 @@ class sbuf_scanner {
 
     void check_margin() {
         if (pos >= sbuf.pagesize ) {
-            throw margin_reached();
+            // throw margin_reached();
+            point = sbuf.bufsize+1;
```

Comment:
It's better than throwing an exception, anyway. Suggest deleting the `margin_reached()` exception type.

Reply:
Deleting the exception now!